components/Chat/ChatVersion.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { View, ViewStyle, StyleSheet, Dimensions} from 'react-native';
+import ThemedText from '../UI/ThemedText';
+
+interface Props {
+  name: string;
+};
+
+const { height, width } = Dimensions.get('window')
```

Comment:
Add types here.

Reply:
I don't know the exact type so i have used 'any' Type